### ISSUE TITLE
Bug: select the right tab when forgot password action is requested 

### ISSUE THREAD

**Issue by AnthonyNahas** [AnthonyNahas/ngx-auth-firebaseui#121]
### Bug Report or Feature Request (mark with an `x`)
```
- [ x] bug report -> please search issues before submitting
- [ ] feature request
```

### Versions
`angular 6.1.7`
`material 6.4.7`

### Repro steps
1. click on forgot password
2. the reset password tab is not selected

### Desired functionality
after requesting to reset the password, the appropriate tab should be selected

**Comment by kjetilhp** [comment]
This bug is back, using latest angular and angular material (8.x)

**Comment by AnthonyNahas** [comment]
@kjetilhp I will check it... are u using the v4 beta version of the library ?

**Comment by kjetilhp** [comment]
@AnthonyNahas I'm using 2.7.2 .. wasn't aware of any beta releases

**Comment by AnthonyNahas** [comment]
@kjetilhp I've just patched the library with the fix of #121 

![rest](https://user-images.githubusercontent.com/20149774/61360438-19e4ff00-a887-11e9-97de-a90ef32a54ff.gif)

[v3.2.4](https://www.npmjs.com/package/ngx-auth-firebaseui) is online 

thank you 🍻
